log-native/src/main/java/org/ballerinalang/stdlib/log/Utils.java

```diff
@@ -18,124 +18,106 @@
 
 package org.ballerinalang.stdlib.log;
 
+import io.ballerina.runtime.api.utils.IdentifierUtils;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BMap;
 import io.ballerina.runtime.api.values.BString;
-import org.ballerinalang.logging.util.BLogLevel;
 
-import java.util.HashMap;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 import java.util.Map;
 
+import static java.lang.System.err;
+
 /**
  * Native function implementations of the log-api module.
  *
  * @since 1.1.0
  */
-public class Utils extends AbstractLogFunction {
-
-    public static final String GLOBAL_PACKAGE_PATH = ".";
-    private static String packagePath = GLOBAL_PACKAGE_PATH;
-    private static Map<String, BLogLevel> loggerLevels = new HashMap<>();
-    private static BLogLevel ballerinaUserLogLevel = BLogLevel.INFO; // default to INFO
+public class Utils {
 
     /**
-     * Prints the log message.
+     * Prints the log message in json format.
      *
-     * @param logLevel log level
      * @param msg log message
-     * @param format output format
      */
-    public static void printExtern(BString logLevel, BString msg, BString format) {
-        switch (BLogLevel.toBLogLevel(logLevel.getValue())) {
-            case DEBUG:
-                logMessage(msg, packagePath,
-                        (pkg, message) -> {
-                            getLogger(pkg).debug(message);
-                        },
-                        format.toString());
-                break;
-            case INFO:
-                logMessage(msg, packagePath,
-                        (pkg, message) -> {
-                            getLogger(pkg).info(message);
-                        },
-                        format.toString());
-                break;
-            case ERROR:
-                logMessage(msg, packagePath,
-                        (pkg, message) -> {
-                            getLogger(pkg).error(message);
-                        },
-                        format.toString());
-                break;
-            case WARN:
-                logMessage(msg, packagePath,
-                        (pkg, message) -> {
-                            getLogger(pkg).warn(message);
-                        },
-                        format.toString());
-                break;
-            default:
-                break;
-        }
+    public static void printJsonExtern(BString msg) {
```

Comment:
Rather than having a separate method. Can't we directly use java interop `java.lang.System.err.println() `

Reply:
Fixed in 3408ba7